sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubsRetryOptions.cs

```diff
@@ -97,8 +97,8 @@ public TimeSpan MaximumDelay
         ///
         /// <value>The default timeout is 60 seconds.</value>
         ///
-        /// <exception cref="ArgumentException">Occurs when the requested delay is negative.</exception>
-        /// <exception cref="ArgumentOutOfRangeException">Occurs when the requested delay is not between <see cref="TimeSpan.Zero" /> and 1 hour (inclusive).</exception>
+        /// <exception cref="ArgumentException">Occurs when the requested timeout is negative.</exception>
```

Comment:
Noticed what appears to be a typo while I was shamelessly copying over the EH docs 😄

Reply:
Yeah.... that would seem to have been forgotten during one of the API renames.  😢